src/bin/merchant/establish.rs

```diff
@@ -318,7 +315,7 @@ async fn establish_channel(
 
     // If the merchant contribution was greater than zero, fund the channel on chain, and await
     // confirmation that the funding has gone through to the required confirmation depth
-    if merchant_deposit.into_inner() > 0 {
+    if !merchant_deposit.is_zero() {
```

Comment:
Just a minor thought, I know this is `u64` so can't technically be negative, but this might be a little easier to read with the `> 0`?

Reply:
Yeah, that is fair. I will add an `is_positive` function to `Merchant/CustomerBalance` so this is less confusing.